tests/qunit/setup-editor.js

```diff
@@ -97,10 +97,61 @@ const elementorConfigSettings = {
 			elementor_lightbox_ui_color_hover: '',
 		},
 	},
-	editorPreferences: {},
+	editorPreferences: {
+		name: 'editorPreferences',
+		panelPage: {
+			title: 'Editor Preferences',
+		},
+		controls: {
+			preferences: {
+				type: 'section',
+				tab: 'settings',
+				label: 'Preferences',
+				name: 'preferences',
+			},
+			ui_theme: {
+				type: 'select',
+				tab: 'settings',
+				section: 'preferences',
+				label: 'UI Theme',
+				description: 'Set light or dark mode, or use Auto Detect to sync it with your OS setting.',
+				default: 'auto',
+				options: {
+					auto: 'Auto Detect',
+					light: 'Light',
+					dark: 'Dark',
+				},
+				name: 'ui_theme',
+			},
+			edit_buttons: {
+				type: 'switcher',
+				tab: 'settings',
+				section: 'preferences',
+				label: 'Editing Handles',
+				description: 'Show editing handles when hovering over the element edit button.',
+				name: 'edit_buttons',
+				default: '',
+			},
+			lightbox_in_editor: {
+				type: 'switcher',
+				tab: 'settings',
+				section: 'preferences',
+				label: 'Enable Lightbox In Editor',
+				default: 'yes',
+				name: 'lightbox_in_editor',
+			},
+		},
+		tabs: {
+			settings: 'Settings',
+		},
+		settings: {
+			ui_theme: 'dark',
+			edit_buttons: '',
+			lightbox_in_editor: 'yes',
+		},
+	},
 };
 
-/* global jQuery */
```

Comment:
Why?

Reply:
Not needed.
![image](https://user-images.githubusercontent.com/10234691/71619865-2c51bb80-2bcf-11ea-8034-35e677bf181e.png)

should be handled like this:
![image](https://user-images.githubusercontent.com/10234691/71619878-49868a00-2bcf-11ea-970d-858a1a1709a3.png)